csrankings-c.csv

```diff
@@ -621,6 +622,7 @@ Chris Clack,University College London,http://www0.cs.ucl.ac.uk/people/C.Clack.ht
 Chris Clifton,Purdue University,https://www.cs.purdue.edu/people/clifton,C_cWJIkAAAAJ
 Chris Crispin-Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb,QQ_dWtcAAAAJ
 Chris De Sa,Cornell University,http://www.cs.cornell.edu/~cdesa,v7EjGHkAAAAJ
+Chris Ding,CUHK-SZ,https://sds.cuhk.edu.cn/teacher/197,q7FfnjgAAAAJ
 Chris Ding,Chinese University of Hong Kong,https://sds.cuhk.edu.cn/en/teacher/197,q7FfnjgAAAAJ

```

Comment:
Notice there are two listings for the same faculty member. Which institution are they at?

Reply:
Prof. Chris Ding is with CUHK-SZ. I corrected the replication